deebot_client/hardware/deebot/4vhygi.py

```
@@ -0,0 +1 @@
+/home/osboxes/Documents/GitHub/client.py/deebot_client/hardware/deebot/p1jij8.py
```

Ruff

No such file or directory (os error 2)

(E902)